practice-one/src/index.html

```diff
@@ -111,9 +111,11 @@ <h1 class="homepage-heading">Products Management</h1>
 
     <div class="modal">
       <div class="modal-container-add">
-        <header class="modal-header">Create new a product</header>
+        <header class="modal-header"></header>
```

Comment:
Why is this empty?

Reply:
When handle edit, add product, the tittle will add